app/commodities/page.client.tsx

```diff
@@ -1,79 +1,194 @@
+      data.includePlanetary
+    }&x=0&y=0&z=0`;
 
-    let submitData: ReqBody = {
-      ...data,
-      commodityId: formatString(data.commodityId.value),
-      minDemand: Number(data.minDemand),
-      minSupply: Number(data.minSupply),
-      referenceLocation: {
-        xcoordinate: 0,
-        ycoordinate: 0,
-        zcoordinate: 0,
-      },
-    };
-    delete submitData.system;
+    try {
+      const commodityReq = await fetch(
+        `/api/v1/trade/locate-commodity/filter?${queryString}`,
+      );
+      const commodityRes = await commodityReq.json();
+      setCommodityResponse(commodityRes);
+    } catch (error) {
+      setFetchError(error as Error);
+    } finally {
+      setIsLoading(false);
+      setSubmitted(true);
+    }
+  };
+
+  // TODO: remove this function after development - aslink87
+  const handleExampleSubmit = async () => {
+    setIsLoading(true);
+    setSubmitted(false);
 
-    // TODO: submit data to backend
-    setTimeout(() => {
-      console.log('submitted ', submitData);
+    try {
+      const commodityReq = await fetch(
+        `${process.env.NEXT_PUBLIC_MOCK_API_URL}/api/v1/trade/commodity`,
+        {
+          cache: 'no-store',
+        },
+      );
+      const commodityRes = await commodityReq.json();
+      setCommodityResponse(commodityRes);
+    } catch (error) {
+      setFetchError(error as Error);
+    } finally {
       setIsLoading(false);
-    }, 2000);
+      setSubmitted(true);
+    }
+  };
+
+  const checkBreakpointBeforeShowingResponse = () => {
+    if (isLarge) {
+      return (
+        <CommodityFormResponse
+          // TODO: remove this slice after pagination/truncate is implemented - aslink87
+          commodityResponse={commodityResponse.slice(0, 20)}
+          isBuying={isBuying}
+        />
+      );
+    }
+    return (
+      <CommodityFormResponseMobile
+        // TODO: remove this slice after pagination/truncate is implemented - aslink87
+        commodityResponse={commodityResponse.slice(0, 20)}
+        isBuying={isBuying}
+      />
+    );
   };
 
   return (
-    <Flex flexDirection="column" gap="24px" width="100%">
-      <Box alignSelf="baseline">
-        <Heading
-          as="h1"
-          size={{ base: 'md', md: 'lg', lg: 'lg' }}
-          marginX={{ base: 'auto', md: '0', lg: '0' }}
-          color={selectColor(isDark, 'accent-text')}
-        >
-          Commodities
-        </Heading>
-        <Heading
-          as="h2"
-          size={{ base: 'xs', md: 'sm', lg: 'sm' }}
-          marginX={{ base: 'auto', md: '0', lg: '0' }}
-        >
-          Find Closest Station to Buy/Sell Commodities
-        </Heading>
-      </Box>
-      <Box
-        borderWidth="2px"
-        borderRadius="9px"
-        borderColor={selectColor(isDark, 'border')}
-        bg={selectColor(isDark, 'accent-bg')}
-        padding="1rem"
+    <Box
+      flex="1"
+      as="main"
+      backgroundImage={`url('/assets/Anaconda_Opacity.svg')`}
```

Comment:
I like the background image, but it does reduce legibility on some form elements, like this.
![image](https://github.com/ed-pilots-network/frontend/assets/50422789/9f714e27-d530-4204-b9da-2c20d0eebd87)
Perhaps we could make the background for these form elements opaque.

Also, if we're going to have ships on every page, maybe we should start picking them based on the context - i.e a Hauler for commodities as opposed to an FDL. That's probably a discussion to have though.

Reply:
Thanks for taking the time to look over this Josh

Reply:
I've merged in the updated Chakra colors and added a background to these forms to resolve this.
![Screenshot 2023-08-29 at 15-30-35 EDPN - Commodities](https://github.com/ed-pilots-network/frontend/assets/67390534/145b3826-8843-4890-8240-bb6681d7e546)